exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/logs/OtlpStdoutLogRecordExporter.java

```diff
@@ -53,8 +58,12 @@ public CompletableResultCode export(Collection<LogRecordData> logs) {
     }
 
     if (wrapperJsonObject) {
-      LogsRequestMarshaler request = LogsRequestMarshaler.create(logs);
-      return jsonWriter.write(request);
+      return new LogReusableDataMarshaler(memoryMode) {
+        @Override
+        public CompletableResultCode doExport(Marshaler exportRequest, int numItems) {
+          return jsonWriter.write(exportRequest);
+        }
+      }.export(logs);
     } else {
       for (ResourceLogsMarshaler resourceLogs : ResourceLogsMarshaler.create(logs)) {
```

Comment:
It doesn't seem to be the case that if `wrapperJsonObject = false`, we can't take advantage of reuseable memory mode. Can we include `wrapperJsonObject` as an argument to `LogReuseableDataMarshaler` and adjust the logic accordingly?

Reply:
that's not so easy, because https://github.com/open-telemetry/opentelemetry-java/blob/715211e98b65c21a2d0ef74d8113f297ece430b0/exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/LowAllocationLogsRequestMarshaler.java#L55 takes a list and I didn't follow what has to be modified in there

Reply:
Ok let's punt on it for now